ARSIS-Unity/Assets/ARSIS/Tools/Caches/BiometricsCache.cs

```diff
@@ -7,9 +7,13 @@ public class BiometricsCache : MonoBehaviour
 {
 
     public BiometricsEvent BiometricsEvent;
+    private static int MINUTES = 5;
+    private static int MAX_ENTRYS = 60*MINUTES;
+    private List<BiometricsEvent> BiometricsList;eee
     public static BiometricsCache BiometricsCacheSingleton { get; private set; }
```

Comment:
Looks like you have a typo here: "eee" after the semicolon

Reply:
I think this should be fixed now, let me know if it's not! :)